src/layouts/Resources.jsx

```diff
@@ -4,18 +4,70 @@ import axios from 'axios';
 import PropTypes from 'prop-types';
 import * as Bluebird from 'bluebird';
 import * as _ from 'lodash';
-import { prettifyResourceFileName } from '../utils';
-import ResourceCard from '../components/ResourceCard';
+import { prettifyResourceFileName, prettifyResourceCategory } from '../utils';
+import ResourceSettingsModal from '../components/ResourceSettingsModal';
 import ResourceCategoryModal from '../components/ResourceCategoryModal';
+import Header from '../components/Header';
+import Sidebar from '../components/Sidebar';
+import elementStyles from '../styles/isomer-cms/Elements.module.scss';
+import contentStyles from '../styles/isomer-cms/pages/Content.module.scss';
+
+// Constants
+const RADIX_PARSE_INT = 10;
+
+const ResourceCard = ({
```

Comment:
As commented above, perhaps page and resource cards can have their own components in separate files

Reply:
Refactored PageCard in commit 7ebfbcf, but kept it in the same file. I think if the component is a stateless function and it isn't too long, we could leave it in the same file. What are your thoughts? 

I agree that if it is a stateful component, we definitely need to move it to a different file like the Modal components.

Reply:
sounds good to me